config/kilted/requirements/docs.ros.org.yaml

```diff
@@ -0,0 +1,2012 @@
+      - name: I was able to follow the documentation.
+      - name: The documentation seemed clear to me.
+      - name: The documentation didn't have any obvious errors.
+  - name: Logging
+    labels:
+      - tutorials
+      - demos
+      - docs
+    description: Check the documentation for the 'Logging' page
+    links:
+      - name: Logging page
+        url: >-
+          https://docs.ros.org/en/kilted/Tutorials/Demos/Logging-and-logger-configuration.html
+    checks:
+      - name: I was able to follow the documentation.
+      - name: The documentation seemed clear to me.
+      - name: The documentation didn't have any obvious errors.
+  - name: Managing nodes with managed lifecycles
+    labels:
+      - tutorials
+      - demos
+      - docs
+    description: Check the documentation for the 'Managing nodes with managed lifecycles' page
+    links:
+      - name: Managing nodes with managed lifecycles page
+        url: https://docs.ros.org/en/kilted/Tutorials/Demos/Managed-Nodes.html
+    checks:
+      - name: I was able to follow the documentation.
+      - name: The documentation seemed clear to me.
+      - name: The documentation didn't have any obvious errors.
+  - name: Using quality-of-service settings for lossy networks
+    labels:
+      - tutorials
+      - demos
+      - docs
+    description: Check the documentation for the 'Using quality-of-service settings for lossy networks' page
+    links:
+      - name: Using quality-of-service settings for lossy networks page
+        url: https://docs.ros.org/en/kilted/Tutorials/Demos/Quality-of-Service.html
+    checks:
+      - name: I was able to follow the documentation.
+      - name: The documentation seemed clear to me.
+      - name: The documentation didn't have any obvious errors.
+  - name: Understanding real-time programming
+    labels:
+      - tutorials
+      - demos
+      - docs
+    description: Check the documentation for the 'Understanding real-time programming' page
+    links:
+      - name: Understanding real-time programming page
+        url: >-
+          https://docs.ros.org/en/kilted/Tutorials/Demos/Real-Time-Programming.html
+    checks:
+      - name: I was able to follow the documentation.
+      - name: The documentation seemed clear to me.
+      - name: The documentation didn't have any obvious errors.
+  - name: Recording and playing back data with rosbag using the ROS 1 bridge
+    labels:
+      - tutorials
+      - demos
+      - docs
+    description: Check the documentation for the 'Recording and playing back data with rosbag using the ROS 1 bridge' page
+    links:
+      - name: Recording and playing back data with rosbag using the ROS 1 bridge page
+        url: >-
+          https://docs.ros.org/en/kilted/Tutorials/Demos/Rosbag-with-ROS1-Bridge.html
+    checks:
+      - name: I was able to follow the documentation.
+      - name: The documentation seemed clear to me.
+      - name: The documentation didn't have any obvious errors.
```

Comment:
Imo we should delete this page from the tutorial since we no longer support ROS 1 or the ROS 1 bridge on 24.04...